src/Prefetch.cpp

```diff
@@ -194,12 +194,18 @@ class InjectPlaceholderPrefetch : public IRMutator {
         Stmt body = mutate(op->body);
 
         if (!prefetch_list.empty() && starts_with(op->name, prefix)) {
+            // Remove ".$n", added by sliding window.
```

Comment:
I still think it would be better for sliding window to rewrite the prefetchdirectives to use the new loop names